data/build_probe_data.py

```diff
@@ -0,0 +1,120 @@
+import tensorflow.compat.v1 as tf
```

Comment:
Python style guide suggests that imports should be ordered .
https://google.github.io/styleguide/pyguide.html (Section 3.13 Imports Formatting)

you can setup `pylint` 
https://google.github.io/styleguide/pyguide.html
to check your code against the Google style guide (some internal tools also help with automatically fixing those issues)